Aztec/Classes/Extensions/UIFont+Traits.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+import UIKit
+
+
+// MARK:- UIFont Traits Helpers
+//
+extension UIFont {
+
+    /// Returns a new instance of the current font with its SymbolicTraits Updated.
+    ///
+    /// - Parameters:
+    ///     - trait: Trait to be updated
+    ///     - enable: Boolean indicating whether we should insert or remove the specified trait.
+    ///
+    /// - Returns: A new UIFont with the same descriptors as the current instance, but with its traits updated, as specified.
+    ///
+    func modifyTrait(_ trait: UIFontDescriptorSymbolicTraits, enable: Bool) -> UIFont {
```

Comment:
Also: nice work... the code in this method should be used [here](https://github.com/wordpress-mobile/WordPress-Aztec-iOS/blob/develop/Example/Example/FormattingDemoController.swift#L148) and [here](https://github.com/wordpress-mobile/WordPress-Aztec-iOS/blob/develop/Example/Example/FormattingDemoController.swift#L106).